catanatron_rust/benches/copy_benchmark.rs

```diff
@@ -0,0 +1,72 @@
+use catanatron_rust::decks::{DevCardDeck, ResourceDeck};
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+
+fn copy_vector_benchmark(c: &mut Criterion) {
+    // Initialize a vector of length 600
+    let vector = vec![0u8; 600];
+
+    c.bench_function("Clone 600-length vector", |b| {
+        let mut iteration_number = 0;
+        b.iter(|| {
+            let mut copied_vector = vector.clone();
+            let index = iteration_number % 600;
+            let random_value = (iteration_number % 256) as u8;
+            copied_vector[index] = random_value;
+            black_box(copied_vector);
+
+            iteration_number += 1;
+        })
+    });
+}
+
+fn copy_array_benchmark(c: &mut Criterion) {
+    // Initialize an array of length 1200
+    let array = [0u8; 600];
+
+    c.bench_function("Copy 600-length array", |b| {
+        let mut iteration_number = 0;
+        b.iter(|| {
+            let mut copied_array = array;
```

Comment:
Is there a missing `.clone()` call here?

i.e. Should this be
`let mut copied_array = array.clone();`
?